.travis.yml

```diff
@@ -6,12 +6,12 @@ rvm:
 sudo: false
 
 install: 
-- gem install jekyll -v 2.5.3
+- bundle install
```

Comment:
it does not, think of this like `cargo build`, it generates and sets up everything. 

Basically,

1. the Gemfile mentions github-pages, [which has a = dependency on jekyll](https://rubygems.org/gems/github-pages)
2. `bundle install` installs all dependencies, which includes jekyll because of this.
3. `bundle exec jekyll build` executes the exact version of `jekyll` that was installed through `bundle install`.